config/dev.exs

```diff
@@ -5,3 +5,7 @@ config :monero,
 
 config :monero, :wallet,
   url: "http://127.0.0.1:18081"
+
+config :monero, :daemon,
+  url: "http://127.0.0.1:28081"
```

Comment:
i think that daemon and wallet binds to 18081 to 18082 ports accordingly

Reply:
Nah, I tested it.

https://github.com/monero-project/monero/blob/4466b6d1b08d827f3bdddb1cab8ca05833b65437/src/cryptonote_config.h#L164